.evergreen/config.yml

```diff
@@ -1799,15 +1825,21 @@ buildvariants:
     tasks:
       - name: ".test !.enterprise-auth !.zlib !.zstd"
 
-  - matrix_name: "tests-36-40-with-zlib-support"
-    matrix_spec: { version: ["3.6", "4.0"], os-ssl-32: "*" }
+  - matrix_name: "tests-36-with-zlib-support"
+    matrix_spec: { version: ["3.6"], os-ssl-32: "*" }
```

Comment:
Specifically here, if 3.6 can run on Ubuntu 18.04, could the OS's just be changed in this matrix which support 3.6+?

Reply:
As mentioned above, SSL setup issues on 3.6 with Ubuntu 18.04 mean we can't combine these 😢 .